examples/qasm.ipynb

```diff
@@ -0,0 +1,394 @@
+{
```

Comment:
<p>Suggestion: you could add</p><p> </p><ul><li>For this demonstration, we already saved a few qasm circuit examples in the directory <code>qasm_files</code>. You can find more examples at [<a href="https://github.com/Qiskit/openqasm" target="_blank">OpenQASM repository</a> Let's start off by reading one of the example:</li><li>Add a section name: QASM Import</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>